joomla-projects/gsoc22_accessibility-plugin
# Made text to speech voice adapt to the website language

### Summary of changes

This pr combined with https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/8 fixes :  https://forum.joomla.org/viewtopic.php?f=808&t=991513&p=3651110


### Testing Instructions

1. Change the site language to any other language other than en-us.
2. Enable text-to-speech in a11y plugin.
3. Check if the voice of the text-to-speech speaker changes to site language.


Note: It is possible that the selected language is not supported by the speechSynthesis API, in that case 'en-US' will be be the default language.

### Actual result BEFORE applying this Pull Request


The voice of the text to speech speaker remains same(en-US) irrespective of the site language.

### Expected result AFTER applying this Pull Request

The voice of the text-to-speech speaker changes to site language.
